### ISSUE TITLE
Remove Whonix 13 TemplateVM packages from Qubes repository

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#4515]
Please remove:

* `qubes-template-whonix-gw`
* `qubes-template-whonix-ws`

(Whonix 13)

Please keep:

* `qubes-template-whonix-gw-14`
* `qubes-template-whonix-ws-14`

Reason: Prevent users from accidentally (re-)installing Whonix 13. Rather fail early than let them go though a lengthy Whonix 13 (jessie based!) upgrade process to then figure out it would have been better to get Whonix 14.

**Comment by esote** [comment]
@andrewdavidwong Once this is done, would [this documentation](https://www.qubes-os.org/doc/supported-versions/#templatevms) need to be updated? (currently we list Whonix 13 as supported)

**Comment by andrewdavidwong** [comment]
> Once this is done, would [this documentation](https://www.qubes-os.org/doc/supported-versions/#templatevms) need to be updated? (currently we list Whonix 13 as supported)

No, because Qubes 3.2 and 4.0 still support Whonix 13. It's just that Whonix no longer supports Whonix 13. (Likewise, Qubes 3.2 and 4.0 still support Fedora 26, but Fedora no longer supports Fedora 26.) Further explanation:

https://www.qubes-os.org/news/2018/10/05/whonix-support-ending-for-qubes-32/#whonix-support-for-qubes-os

**Comment by andrewdavidwong** [comment]
Since we keep EOL versions of other [template packages](https://ftp.qubes-os.org/repo/yum/r3.2/templates-itl/rpm/) (e.g., Fedora, Debian), perhaps we should keep the Whonix 13 template packages but just rename them to the new naming convention, i.e.:

- `qubes-template-whonix-gw-13`
- `qubes-template-whonix-ws-13`

This should prevent most users from accidentally installing 13 without making 13 wholly inaccessible.

**Comment by adrelanos** [comment]
Sure. Not my preferred solution (to cling on Whonix 13 seems a lost
cause in strive of perfection on applying principles without any user
request or positive gain from it) but of course much better than nothing. :)

**Comment by andrewdavidwong** [comment]
Can this be closed?

**Comment by adrelanos** [comment]
No report of any action done reported. Can only be done by @marmarek as far as I know.
